api/v1alpha1/submariner_types.go

```diff
@@ -228,6 +232,11 @@ type SubmarinerStatus struct {
 
 	AirGappedDeployment bool `json:"airGappedDeployment,omitempty"`
 
+	// Is the cluster a hosted cluster.
+	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Hosted Cluster"
+	// +operator-sdk:csv:customresourcedefinitions:type=status,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+	HostedCluster bool `json:"hostedCluster,omitempty"`
```

Comment:
This should be set by the controller along with the other status fields.

Reply:
Done.